lib/proxy_domain.js

```diff
@@ -6,29 +6,34 @@ const http = require("http");
 const Router = require("express").Router;
 const passport = require("passport");
 const rp = require("request-promise-native");
-const crypto = require('crypto');
+const crypto = require("crypto");
 const LocalStrategy = require("passport-local-token").Strategy;
 const GitHubStrategy = require("passport-github").Strategy;
 const GoogleStrategy = require("passport-google-oauth").OAuth2Strategy;
 
+const verifyInclusion = (valids, checks) => {
+  return checks.some(check => valids.include(check));
```

Comment:
```suggestion
  return checks.some(check => valids.includes(check));
```

Gotta have that `s` on `includes`